boto/awslambda/__init__.py

```diff
@@ -28,13 +30,10 @@ def regions():
     :rtype: list
     :return: A list of :class:`boto.regioninfo.RegionInfo`
     """
-    from boto.awslambda.layer1 import AWSLambdaConnection
```

Comment:
I don't remember why.  I was going to guess to avoid importing all of boto including connection classes on an `import boto`, but the various `boto.connect_<service>`_ functions in `boto/__init__.py` import the connection classes inline, so these are probably unnecessary.

However, my previous stance here still remains for boto which is to be conservative in the changes unless there's a really good reason not to do so vs. making changes that seem harmless but not actually knowing for sure.